src/_variables.scss

```diff
@@ -26,3 +26,9 @@ $c2lc-hover-indicator-width: 0.1rem;
 
 $c2lc-action-panel-width: 16rem;
 $c2lc-action-panel-height: 5rem;
+
+/* Scene */
+
+$c2lc-scene-tile-num-row: 5;
+$c2lc-scene-tile-num-column: 9;
+$c2lc-scene-tile-width: 100;
```

Comment:
As we are using "tile" for the program actions, I think using another term for these variable names would be clearer, such as "grid-cell".